purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1237,34 +1237,35 @@ class Purchases internal constructor(
     }
 
     private fun getSkuDetails(
-        skus: Set<String>,
+        productIds: Set<String>,
         onCompleted: (HashMap<String, StoreProduct>) -> Unit,
         onError: (PurchasesError) -> Unit
     ) {
         billing.querySkuDetailsAsync(
             ProductType.SUBS,
-            skus,
-            { subscriptionsSKUDetails ->
-                val detailsByID = HashMap<String, StoreProduct>()
-                val inAPPSkus =
-                    skus - subscriptionsSKUDetails
-                        .map { details -> details.sku to details }
```

Comment:
i personally found the chaining here a little hard to follow, so i pulled them out and made some more variables. open to undoing if it doesn't seem worth the tradeoff